### ISSUE TITLE
Quill editor toolbar stops disappearing after a certain amount of use

### ISSUE THREAD

**Issue by cydrobolt** [quilljs/quill#788]
We're using Quill for a full-page editor app that creates a new Quill editor for each "content block" aka each new paragraph. Each Quill editor offers a toolbar with some common functions (i.e link, bold, italics, etc). However, after some use, around half an hour, the toolbars stop disappearing. 

I'm thinking it could be possibly a memory leak from editor objects & JS listeners not being cleaned up as the user creates more Quill editors, but I'm not certain. This mainly seems to happen on Windows Chrome. Users on Mac or Linux haven't been able to reproduce the bug to the same extent.

In short timespans or with a small number of editors, I was unable to reproduce this issue.

The toolbar's disappearing is bound to

`this.quill.on('selection-change', this.handleSelectionChange);`

**Steps for Reproduction**

1. Create multiple Quill editors
2. Use the Quill editors over half an editor
3. See frozen toolbar

**Expected behavior**: toolbar should disappear

**Actual behavior**: toolbar stays floating

**Platforms**: Chrome 53 on Windows 8

**Version**: v0.20.0

**Comment by jhchen** [comment]
If you're using 0.20, you actually need to call `quill.destroy()` to free all the listeners. This functionality was added in a PR but never got added to the 0.20 docs. This is no longer necessary in 1.0.
